CONTRIBUTING.md

```diff
@@ -7,6 +7,7 @@ help the maintainer to fix the bug that you've discovered quicker:
 
 * [Use the PHP Driver Jira](https://datastax-oss.atlassian.net/projects/PHP) to report all issues and bugs.
 * Include the version of the Driver, PHP and Cassandra or DSE in your Jira ticket.
+ * Include the Driver dependency versions as well (e.g. libuv, C/C++ Driver, ...etc)
```

Comment:
Heh, I've already fixed it, just realized you were OOO

On Fri, May 22, 2015 at 3:23 PM, Fero <notifications@github.com> wrote:

> In CONTRIBUTING.md
> <https://github.com/datastax/php-driver/pull/31#discussion_r30935196>:
>
> > @@ -7,6 +7,7 @@ help the maintainer to fix the bug that you've discovered quicker:
> >
> >  * [Use the PHP Driver Jira](https://datastax-oss.atlassian.net/projects/PHP) to report all issues and bugs.
> >  * Include the version of the Driver, PHP and Cassandra or DSE in your Jira ticket.
> > + * Include the Driver dependency versions as well (e.g. libuv, C/C++ Driver, ...etc)
>
> This was intentional as it is related to the related version information
> bullet. Would you still prefer it at the same level?
> … <#14d7db98a05c3a55_>
> On Friday, May 22, 2015, Bulat Shakirzyanov <notifications@github.com>
> wrote: In CONTRIBUTING.md <
> https://github.com/datastax/php-driver/pull/31#discussion_r30914344>: >
> @@ -7,6 +7,7 @@ help the maintainer to fix the bug that you've discovered
> quicker: > > * [Use the PHP Driver Jira](
> https://datastax-oss.atlassian.net/projects/PHP) to report all issues and
> bugs. > * Include the version of the Driver, PHP and Cassandra or DSE in
> your Jira ticket. > + * Include the Driver dependency versions as well
> (e.g. libuv, C/C++ Driver, ...etc) please remove the extra space before * —
> Reply to this email directly or view it on GitHub <
> https://github.com/datastax/php-driver/pull/31/files#r30914344>.
>  -- Fero [image: datastax_logo.png] Michael Fero Software Engineer in
> Test | +1.407.702.8321 | michael.fero@datastax.com [image: linkedin.png] <
> https://www.linkedin.com/company/datastax> [image: facebook.png] <
> https://www.facebook.com/datastax> [image: twitter.png] <
> https://twitter.com/datastax> [image: g+.png] <
> https://plus.google.com/+Datastax/about> <
> http://feeds.feedburner.com/datastax> <https://github.com/datastax/>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datastax/php-driver/pull/31/files#r30935196>.
>



-- 
*Bulat Shakirzyanov* | Software Alchemist

*a: *about.me/avalanche123
*e:* mallluhuct@gmail.com